versions/models.py

```diff
@@ -57,6 +57,16 @@ def get_slug(self):
     def display_name(self):
         return self.name.replace("boost-", "")
 
+    @cached_property
+    def non_beta_slug(self):
+        """Return the slug without any beta suffix.
+
+        Example:
+        - "boost-1.75.0.beta1" --> "boost-1-75-0"
+        - "develop" --> "develop"
+        """
+        return self.slug.split("-beta")[0]
```

Comment:
`.rstrip('-beta')`

Reply:
That won't work since the suffix is `beta1`, `beta2`, etc.